config/config.example.yml

```diff
@@ -15,6 +15,29 @@ db:
   port: 5432
   dbname: invidious
 
+  ##
+  ## Granular pooling behavior controls
+  ## The following is entirely optional.
+  ##
+
+  ## Initial amount of connections in the pool 
+  #initial_pool_size:
+
+  ## Maximum amount of connections in the pool. 0 is unlimited.
+  #max_pool_size: 0
+
+  ## Amount of connections when idle
+  #max_idle_pool_size: 1 
+
+  ## Amount of seconds to wait if the connection pool is full, and a connection is unavailable. 
+  #checkout_timeout: 5 
+
+  ## Amount of retries when a connection is either lost or can't be established.
+  #retry_attempts: 1
```

Comment:
Should we bump this number to a higher value?

Reply:
Most of the reported issues are related to `checkout_timeout` and not `retry_attempts` so I don't think it'd help that much if we change that. But just in case, I've went ahead and increased it to three, which should hopefully make errors like `DB::PoolRetryAttemptsExceeded` (which is already quite rare) nonexistent.